src/sdk/config_fs.js

```diff
@@ -438,6 +438,25 @@ class ConfigFS {
         return path.join(this.old_accounts_dir_path, this.json(account_name));
     }
 
+    /**
+     * stat_account_config_file will return the stat output on account config file
+     * please notice that stat might throw an error - you should wrap it with try-catch and handle the error
+     * Note: access_key type of anonymous_access_key is a symbol, otherwise it is a string (not SensitiveString)
+     * @param {Symbol|string} access_key
+     * @returns {Promise<nb.NativeFSStats>}
+     */
+    stat_account_config_file(access_key) {
+        let path_for_account_or_user_config_file;
+        if (typeof access_key === 'symbol') { // anonymous account case
```

Comment:
add a check that it's the anonymous symbol

Reply:
I added it, thanks.